### DISCUSSION TITLE
Disable social card

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6713] [Q&A] [answered]
Is there some syntax like this to disable social card from being generated on this page?
```
---
icon: bla bla bla
no_social_card: true
---
```

**Comment by kamilkrzyskow** [comment, marked as answer]
The community version of the Social plugin doesn't provide this functionality, however the Insiders version does. 
You can either patch the community version to make it work as you want or support the project to get access to Insiders.
It will take some more time before the insiders version will be merged with the community version https://squidfunk.github.io/mkdocs-material/insiders/#24000-blockpaprika
